### DISCUSSION TITLE
Too many requests to RPC Endpoint after upgrading to v6.8.1-beta

### DISCUSSION THREAD

**Discussion by MaxTeiger** [blockscout/blockscout#11192] [Q&A]
Hello ! 

I updated my blockscout from : 

- Backend: [v5.2.1-beta](https://github.com/blockscout/blockscout/tree/v5.2.1-beta)
- Frontend: [v1.11.1](https://github.com/blockscout/frontend/tree/v1.11.1)
to
- Backend: [v6.8.1-beta](https://github.com/blockscout/blockscout/tree/v6.8.1-beta)
- Frontend: [account-v2.4](https://github.com/blockscout/frontend/tree/account-v2.4)

And since then, the number of requests to my RPC endpoint skyrocketed :
<img width="471" alt="image" src="https://github.com/user-attachments/assets/d294cc60-f4b8-421a-9795-5ba695bfbab1">

We can note that I also switched from a Helm deployed Blockscout to a docker-compose deployed blockscout, for simplicity. 

- Do you know where the problem might come from ? (`docker-compose` vs `helm` deployment? version upgrade?)
- Is there a way to reduce the number of requests made to the RPC endpoint without downgrading the version ? (I'm not even sure it's possible to downgrade)

**Comment by nikitosing** [comment]
@MaxTeiger Hey, do you have any statistic with requests types?
also would be useful to get info about your blockscout settings
